docs/upstream-differences.md

```diff
@@ -35,3 +35,55 @@ See https://github.com/openshift/installer/compare/release-4.6...mjudeikis:relea
   communicate with the cluster.
 
 * ARO runs a dnsmasq service on the nodes through the use of a machineconfig to resolve api-int and *.apps domains on the node locally allowing for custom DNS configured on the VNET.
+
+# Introducing new OCP release into ARO RP
+
+## Update installer fork
+
+To bring new OCP release branch into ARO installer fork:
+
+1. Check git diff between the target release branch release-X.Y and previous one release-X.Y-1
+   to see if any resources changed and/or architecture changed.
+   These changes might require more modifications on ARO-RP side later on.
+1. Create a new release-X.Y-azure branch in the ARO installer fork from upstream release-X.Y branch.
+1. Cherry-pick all commits from a previous release branch into new the branch & fix conflicts.
```

Comment:
```suggestion
1. Cherry-pick all commits from the previous release-X.Y-1 branch into the new one & fix conflicts.
```

I tried to clarify this sentence - did I understand it correctly?

Reply:
@nilsanderselde thanks a lot of these suggestions. You got everything correct. I'll update PR shortly.

The only slight correction would be: in this specific suggestion it will be more accurate to say `release-X.Y-1-azure` instead of `release-X.Y-1` becuase here I'm referring to our custom branch in our fork rather than upstream release branch.  I'll make sure that I include this.